frontend/src/container/PlannedDowntime/PlannedDowntimeForm.tsx

```diff
@@ -309,62 +305,65 @@ export function PlannedDowntimeForm(
 				</Form.Item>
 				<Form.Item
 					label="Repeats every"
-					name="recurrenceSelect"
-					required={false}
+					name={['recurrence', 'repeatType']}
```

Comment:
can you help me understand why these two names here ?

Reply:
This is required for nested approach, like this would be treated as `recurrence.repeatType` as this is format supported by payload and list call also